roles/ceph-mon/tasks/openstack_config.yml

```diff
@@ -1,11 +1,7 @@
 ---
 - name: create openstack pool
   command: ceph --cluster {{ cluster }} osd pool create {{ item.name }} {{ item.pg_num }}
-  with_items:
-    - "{{ openstack_glance_pool }}"
-    - "{{ openstack_cinder_pool }}"
-    - "{{ openstack_nova_pool }}"
-    - "{{ openstack_cinder_backup_pool }}"
+  with_items: openstack_pools
```

Comment:
Hum I think with Ansible v2 this needs to be `"{{ openstack_pools }}"`.
What's your Ansible version?